svc-lighthouse-api/src/main/java/gov/va/vro/abddataaccess/service/FhirClient.java

```diff
@@ -219,8 +219,20 @@ public List<AbdBloodPressure> getPatientBloodPressures(List<BundleEntryComponent
     List<AbdBloodPressure> result = new ArrayList<>();
     for (BundleEntryComponent entry : entries) {
       Observation resource = (Observation) entry.getResource();
-      AbdBloodPressure summary = FieldExtractor.extractBloodPressure(resource);
-      result.add(summary);
+      AbdBloodPressure bpReading = FieldExtractor.extractBloodPressure(resource);
+      if ((bpReading.getDiastolic() == null) && (bpReading.getSystolic() == null)) {
+        continue; // skip it if both systolic and diastolic values are missing.
+      }
+      // Set default systolic / diastolic blood pressure value if one of them not exist.
+      if (bpReading.getDiastolic() == null) {
+        bpReading.setDiastolic(
+            FieldExtractor.getDefaultBpMeasurement(FieldExtractor.BP_MEASURE.DIASTOLIC));
+      }
+      if ((bpReading.getSystolic() == null) && (bpReading.getDiastolic() != null)) {
```

Comment:
&& (bpReading.getDiastolic() != null) is not needed and makes code harder to read.